inlang/source-code/paraglide/paraglide-js/src/compilerV2/compileBundle.test.ts

```diff
@@ -0,0 +1,159 @@
+import { it, expect } from "vitest"
+import { compileBundle } from "./compileBundle.js"
+import { createBundle, createMessage } from "@inlang/sdk2"
+
+it("should compile a bundle", () => {
+	const bundleId = "my_bundle"
+	const bundle = createBundle({
+		id: bundleId,
+		messages: [
+			createMessage({
+				bundleId,
+				locale: "en",
+				text: "Hello World!",
+			}),
+			createMessage({
+				bundleId,
+				locale: "de",
+				text: "Hallo Welt!",
+			}),
+		],
+	})
+
+	const output = compileBundle(bundle, {
+		en: undefined,
+		de: "en",
+	})
+	expect(output).toMatchInlineSnapshot(`
```

Comment:
Why did you move to snapshot testing instead of the rollup code testing setup we once had? 

- snapshot change every time JSdoc, ids, etc are updated, that seems like a lot of hassle
- calling the message functions and comparing the output seems much easier

Reply:
I switched back to the bundling based tests. 

I prefer snapshot tests for developing since they do a good job at showing the output. The bundling based tests are more robust once you have something that works